pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md

```diff
@@ -20,10 +20,11 @@ Before starting the Konvoy installation, verify that you have:
 - [kubectl][install_kubectl] for interacting with the running cluster.
 - A valid GCP account with [credentials configured](https://github.com/kubernetes-sigs/cluster-api-provider-gcp).
 
-# Create a Cluster in Google Cloud Platform using clusterctl
+# Create a Cluster in Google Cloud Platform 
 <!NEED CONFIRMATION OF STEPS PRE-RELEASE>
 ## Setup your gcloud CLI
 
+The following steps will be performed using clusterctl.
```

Comment:
wild that this isn't in the accept.txt manifest yet, I can add it super quick

Reply:
Well honestly, this is what was in Dimitri's notes, so I better verify this before I merge.

Reply:
So yeah no, that was backend stuff, not customer facing, so I removed clusterctl reference.